lib/src/ui/list_screen/location_model.dart

```diff
@@ -0,0 +1,14 @@
+class LocationModel {
+  final String city;
+  final String street;
+  final String firstTurnOffStarts;
+  final String firstTurnOffEnds;
+  final String secondTurnOffStarts;
+  final String secondTurnOffEnds;
+  final DateTime date;
+
+  // ignore: sort_constructors_first
+  LocationModel(this.city, this.street, this.firstTurnOffStarts, this.firstTurnOffEnds, this.secondTurnOffStarts,
+      this.secondTurnOffEnds, this.date);
+}
+
```

Comment:
prepare it to null safety
complete example for all of your PR -> https://gist.github.com/56268835d6a35be0c3452e56f68f813b.git